composer.json

```diff
@@ -61,7 +61,12 @@
     },
     "scripts": {
         "phpstan": "phpstan analyse",
-        "fix-code-style": "php-cs-fixer fix"
+        "fix-code-style": "php-cs-fixer fix",
+        "build": [
```

Comment:
https://github.com/box-project/box/blob/main/doc/installation.md#shivammathursetup-php-github-actions

Reply:
it's there also for running it locally 😞